velox/duckdb/conversion/DuckConversion.h

```diff
@@ -24,7 +24,7 @@ class variant;
 
 namespace facebook::velox::duckdb {
 
-::duckdb::LogicalType fromVeloxType(TypeKind kind);
+::duckdb::LogicalType fromVeloxType(const TypePtr& type);
```

Comment:
not related to this PR, but looks like we're missing header comments for these functions.

Reply:
Fixed in #2773